gen/src/main/java/com/softlayer/api/gen/MetaConverter.java

```diff
@@ -76,13 +72,14 @@ public String getPackageName(String typeName) {
             typeName = typeName.substring(10);
         }
         String[] pieces = typeName.split("_");
-        // Skip the last one, it's the class name
-        for (int i = 0; i < pieces.length - 1; i++) {
-            String piece = pieces[i].toLowerCase();
-            String replacement = keywordReplacements.get(piece);
-            piece = replacement != null ? replacement : piece;
-            pkg.append('.').append(getValidJavaIdentifier(piece));
-        }
+        Arrays.stream(pieces)
+                .limit(pieces.length - 1) // Skip the last, it's the class name
+                .map(String::toLowerCase)
+                .map(piece -> {
+                    String replacement = keywordReplacements.get(piece);
+                    return replacement != null ? replacement : piece;
+                })
+                .forEach(piece -> pkg.append('.').append(getValidJavaIdentifier(piece)));
```

Comment:
From one iteration to three...doesn't really matter, but strange to see the change just for change-sake

Reply:
You caught me  :) this was my excuse to play around with streams. I'll revert this.

Reply:
Changed this back.